[0.418s] Invoking command in '/home/vincent/gz/ws_inspect/src/inspection_mission': AMENT_PREFIX_PATH=/home/vincent/gz/ws_inspect/install/inspection_mission:${AMENT_PREFIX_PATH} CONDA_PROMPT_MODIFIER=(ros2) PATH=/home/vincent/gz/ws_inspect/install/inspection_mission/bin:${PATH} PYTEST_ADDOPTS=--tb=short --junit-xml=/home/vincent/gz/ws_inspect/build/inspection_mission/pytest.xml --junit-prefix=inspection_mission -o cache_dir=/home/vincent/gz/ws_inspect/build/inspection_mission/.pytest_cache PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=/home/vincent/gz/ws_inspect/build/inspection_mission/src:/home/vincent/gz/ws_inspect/install/inspection_mission/lib/python3.12/site-packages:${PYTHONPATH} /usr/bin/python3 -m pytest
[1.199s] ============================= test session starts ==============================
[1.199s] platform linux -- Python 3.12.3, pytest-7.4.4, pluggy-1.4.0
[1.199s] cachedir: /home/vincent/gz/ws_inspect/build/inspection_mission/.pytest_cache
[1.199s] rootdir: /home/vincent/gz/ws_inspect/src/inspection_mission
[1.199s] plugins: launch-testing-ros-0.26.9, ament-pep257-0.17.3, ament-lint-0.17.3, launch-testing-3.4.6, ament-copyright-0.17.3, ament-xmllint-0.17.3, ament-flake8-0.17.3, colcon-core-0.20.0, cov-4.1.0
[1.199s] collected 0 items / 1 error
[1.199s] 
[1.199s] ==================================== ERRORS ====================================
[1.199s] ________________________ ERROR collecting test session _________________________
[1.199s] /usr/lib/python3/dist-packages/pluggy/_hooks.py:501: in __call__
[1.199s]     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
[1.199s] /usr/lib/python3/dist-packages/pluggy/_manager.py:119: in _hookexec
[1.199s]     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
[1.199s] /usr/lib/python3/dist-packages/_pytest/python.py:215: in pytest_collect_file
[1.199s]     module: Module = ihook.pytest_pycollect_makemodule(
[1.200s] /usr/lib/python3/dist-packages/_pytest/config/compat.py:66: in fixed_hook
[1.200s]     return hook(**kw)
[1.200s] /usr/lib/python3/dist-packages/pluggy/_hooks.py:501: in __call__
[1.200s]     return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
[1.200s] /usr/lib/python3/dist-packages/pluggy/_manager.py:119: in _hookexec
[1.200s]     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
[1.200s] /opt/ros/jazzy/lib/python3.12/site-packages/launch_testing/pytest/hooks.py:193: in pytest_pycollect_makemodule
[1.200s]     entrypoint = find_launch_test_entrypoint(path)
[1.200s] /opt/ros/jazzy/lib/python3.12/site-packages/launch_testing/pytest/hooks.py:183: in find_launch_test_entrypoint
[1.200s]     module = import_path(path, root=None)
[1.200s] /usr/lib/python3/dist-packages/_pytest/pathlib.py:567: in import_path
[1.200s]     importlib.import_module(module_name)
[1.201s] /usr/lib/python3.12/importlib/__init__.py:90: in import_module
[1.201s]     return _bootstrap._gcd_import(name[level:], package, level)
[1.201s] <frozen importlib._bootstrap>:1387: in _gcd_import
[1.201s]     ???
[1.201s] <frozen importlib._bootstrap>:1360: in _find_and_load
[1.201s]     ???
[1.201s] <frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
[1.201s]     ???
[1.201s] <frozen importlib._bootstrap>:935: in _load_unlocked
[1.201s]     ???
[1.201s] /usr/lib/python3/dist-packages/_pytest/assertion/rewrite.py:186: in exec_module
[1.201s]     exec(co, module.__dict__)
[1.201s] tests/test_mission_node.py:11: in <module>
[1.202s]     from inspection_mission.mission_node import MissionNode
[1.202s] src/inspection_mission/mission_node.py:22: in <module>
[1.202s]     from rclpy.action import ActionClient, ClientGoalHandle
[1.202s] E   ImportError: cannot import name 'ClientGoalHandle' from 'rclpy.action' (/opt/ros/jazzy/lib/python3.12/site-packages/rclpy/action/__init__.py)
[1.202s] - generated xml file: /home/vincent/gz/ws_inspect/build/inspection_mission/pytest.xml -
[1.202s] =========================== short test summary info ============================
[1.202s] ERROR  - ImportError: cannot import name 'ClientGoalHandle' from 'rclpy.actio...
[1.202s] !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
[1.203s] =============================== 1 error in 0.41s ===============================
[1.266s] Invoked command in '/home/vincent/gz/ws_inspect/src/inspection_mission' returned '2': AMENT_PREFIX_PATH=/home/vincent/gz/ws_inspect/install/inspection_mission:${AMENT_PREFIX_PATH} CONDA_PROMPT_MODIFIER=(ros2) PATH=/home/vincent/gz/ws_inspect/install/inspection_mission/bin:${PATH} PYTEST_ADDOPTS=--tb=short --junit-xml=/home/vincent/gz/ws_inspect/build/inspection_mission/pytest.xml --junit-prefix=inspection_mission -o cache_dir=/home/vincent/gz/ws_inspect/build/inspection_mission/.pytest_cache PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=/home/vincent/gz/ws_inspect/build/inspection_mission/src:/home/vincent/gz/ws_inspect/install/inspection_mission/lib/python3.12/site-packages:${PYTHONPATH} /usr/bin/python3 -m pytest
